{"filter":false,"title":"my_package_launch_file.launch","tooltip":"/my_package/launch/my_package_launch_file.launch","ace":{"folds":[],"scrolltop":0,"scrollleft":4,"selection":{"start":{"row":4,"column":9},"end":{"row":4,"column":9},"isBackwards":false},"options":{"guessTabSize":true,"useWrapMode":false,"wrapToView":true},"firstLineState":0},"hash":"d1ac5354be59699f5135bbcb7db2453493aea574","undoManager":{"mark":11,"position":11,"stack":[[{"start":{"row":0,"column":0},"end":{"row":4,"column":9},"action":"insert","lines":["<launch>","    <!-- My Package launch file -->","    <node pkg=\"my_package\" type=\"simple.py\" name=\"ObiWan\"  output=\"screen\">","    </node>","</launch>"],"id":1}],[{"start":{"row":2,"column":33},"end":{"row":2,"column":39},"action":"remove","lines":["simple"],"id":2},{"start":{"row":2,"column":33},"end":{"row":2,"column":55},"action":"insert","lines":["my_package_launch_file"]}],[{"start":{"row":2,"column":33},"end":{"row":2,"column":55},"action":"remove","lines":["my_package_launch_file"],"id":3},{"start":{"row":2,"column":33},"end":{"row":2,"column":55},"action":"insert","lines":["simple_topic_publisher"]}],[{"start":{"row":2,"column":66},"end":{"row":2,"column":72},"action":"remove","lines":["ObiWan"],"id":4},{"start":{"row":2,"column":66},"end":{"row":2,"column":67},"action":"insert","lines":["M"]}],[{"start":{"row":2,"column":67},"end":{"row":2,"column":68},"action":"insert","lines":["o"],"id":5}],[{"start":{"row":2,"column":68},"end":{"row":2,"column":69},"action":"insert","lines":["v"],"id":6}],[{"start":{"row":2,"column":69},"end":{"row":2,"column":70},"action":"insert","lines":["e"],"id":7}],[{"start":{"row":2,"column":70},"end":{"row":2,"column":71},"action":"insert","lines":["R"],"id":8}],[{"start":{"row":2,"column":71},"end":{"row":2,"column":72},"action":"insert","lines":["o"],"id":9}],[{"start":{"row":2,"column":72},"end":{"row":2,"column":73},"action":"insert","lines":["b"],"id":10}],[{"start":{"row":2,"column":73},"end":{"row":2,"column":74},"action":"insert","lines":["o"],"id":11}],[{"start":{"row":2,"column":74},"end":{"row":2,"column":75},"action":"insert","lines":["t"],"id":12}]]},"timestamp":1537435866000}